spec/controllers/user_tips_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe UserTipsController do
+  let(:user) { create(:user) }
+  let(:tip) { Tip.create }
+  let(:user_tip_params) { {user_id: user, tip_id: tip, result: 'Liked', feedback: 'This tip was good'} }
+  before(:each) { sign_in user }
+
+  describe "POST #create" do
+      it 'saves the new UserTip in the database' do
+        expect { post :create, user_tip: user_tip_params }.to change{
+          UserTip.count
+        }.by(1)
+        uploaded_user_tip = UserTip.last
+        expect(uploaded_user_tip.feedback).to_not be_nil
+        expect(uploaded_user_tip.user).to eq user
+      end
```

Comment:
I would write tests for the other user tip parameters (result & tip_id). 

 I would also rewrite line 15 to test uploaded_user_tip.feedback == feedback (if you use my suggestion on line 6.  Otherwise it would be uploaded_user_tip.feedback == "this tip is good").  Your test for feedback will only ensure that it's not nil.  It's better to write a test to assert the value of feedback will be correct.